pkg/agent/const.go

```diff
@@ -57,25 +57,23 @@ const (
 	kubernetesCISScript               = "linux/cloud-init/artifacts/cis.sh"
 	kubernetesHealthMonitorScript     = "linux/cloud-init/artifacts/health-monitor.sh"
 	// kubernetesKubeletMonitorSystemdTimer= "linux/cloud-init/artifacts/kubelet-monitor.timer" // TODO enable,
-	kubernetesKubeletMonitorSystemdService    = "linux/cloud-init/artifacts/kubelet-monitor.service"
-	kubernetesDockerMonitorSystemdTimer       = "linux/cloud-init/artifacts/docker-monitor.timer"
-	kubernetesDockerMonitorSystemdService     = "linux/cloud-init/artifacts/docker-monitor.service"
-	kubernetesContainerdMonitorSystemdTimer   = "linux/cloud-init/artifacts/containerd-monitor.timer"
-	kubernetesContainerdMonitorSystemdService = "linux/cloud-init/artifacts/containerd-monitor.service"
```

Comment:
Only lines 63 and 64 - 
init/artifacts/containerd-monitor.service"
init/artifacts/containerd-monitor.timer"

Were deleted, but it shifted everything else a couple of spaces to the right